cmd/keycloakb/keycloak_bridge.go

```diff
@@ -1191,7 +1200,7 @@ func main() {
 		var route = mux.NewRouter()
 
 		// Version.
-		route.Handle("/", http.HandlerFunc(commonhttp.MakeVersionHandler(keycloakb.ComponentName, ComponentID, keycloakb.Version, Environment, GitCommit)))
+		route.Handle("/", commonhttp.MakeVersionHandler(keycloakb.ComponentName, ComponentID, keycloakb.Version, Environment, GitCommit))
```

Comment:
Why we can now remove it ? is it an "error" from the beginning or a new feature from http ?